x-pack/qa/security-tools-tests/src/test/java/org/elasticsearch/xpack/security/enrollment/tool/CreateEnrollmentTokenToolTests.java

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Same q about FIPS mode

Reply:
There is nothing in the tests themselves that is not FIPS compliant, but there is no need running these in FIPS mode since we don't support the enrollment process in FIPS mode, I'll mute!